src/main/java/it/gov/pagopa/reward/notification/utils/Utils.java

```diff
@@ -110,6 +112,24 @@ public static RefundType getRefundType(RewardsNotification notification) {
         }
     }
 
+    public static BeneficiaryType getBeneficiaryType(String initiativeRewardType) {
```

Comment:
change initiativeRewardType on the DTO in order to deserialize it as a the enum InitiativeRewardType, instead of using String and then to be force to perform this operation